livekit_room.proto

```diff
@@ -37,6 +37,12 @@ service RoomService {
 
   // Subscribes or unsubscribe a participant from tracks. Requires `roomAdmin`
   rpc UpdateSubscriptions(UpdateSubscriptionsRequest) returns (UpdateSubscriptionsResponse);
+
+  // Begins recording a room
+  rpc RecordRoom(RecordRoomRequest) returns (RecordingResponse);
```

Comment:
are these required? or another way of recording?

Reply:
another way, if you want to start it late or end it early